src/frontend/components/DashboardVideoLiveJitsi/index.spec.tsx

```diff
@@ -202,4 +210,91 @@ describe('<DashboardVideoLiveJitsi />', () => {
       expect.any(String),
     );
   });
+
+  it('calls setCanStartLive when role changes', () => {
+    const video = videoMockFactory({
+      live_info: {
+        medialive: {
+          input: {
+            endpoints: [
+              'rtmp://1.2.3.4:1935/stream-key-primary',
+              'rtmp://4.3.2.1:1935/stream-key-secondary',
+            ],
+          },
+        },
+        jitsi: {
+          domain: 'meet.jit.si',
+          external_api_url: 'https://meet.jit.si/external_api.js',
+          config_overwrite: {},
+          interface_config_overwrite: {},
+        },
+      },
+      live_state: liveState.RUNNING,
+      live_type: LiveModeType.JITSI,
+    });
+    global.JitsiMeetExternalAPI = mockJitsi;
+    const mockCanStartLive = jest.fn();
+
+    render(
+      <DashboardVideoLiveJitsi
+        video={video}
+        setCanStartLive={mockCanStartLive}
+      />,
+    );
+
+    expect(mockCanStartLive).not.toHaveBeenCalled();
+
+    // simulates moderator role granted
+    dispatch('participantRoleChanged', {
+      role: 'moderator',
+    });
+
+    expect(mockCanStartLive).toHaveBeenCalledWith(true);
+
+    // simulates user is no more moderator
+    dispatch('participantRoleChanged', {
+      role: 'participant',
+    });
+
+    expect(mockCanStartLive).toHaveBeenCalledWith(true);
```

Comment:
```suggestion
    expect(mockCanStartLive).toHaveBeenCalledWith(false);
```

Do I misunderstand or should this be false? Also reset the mock or check the last call to make sure we're not re-checking the previous call?

Reply:
Good catch !